km_api/know_me/views.py

```diff
@@ -155,6 +146,53 @@ def perform_create(self, serializer):
         return serializer.save(user=self.request.user)
 
 
+class ListEntryListView(generics.ListCreateAPIView):
+    """
+    View for listing and creating list entries.
+    """
+    filter_backends = (filters.ListEntryFilterBackend,)
+    permission_classes = (DRYPermissions,)
+    queryset = models.ListEntry.objects.all()
+    serializer_class = serializers.ListEntrySerializer
+
+    def perform_create(self, serializer):
+        """
+        Create a new list entry for the given list content.
+
+        Args:
+            serializer:
+                The serializer containing the data used to create the
+                new entry.
+
+        Returns:
+            The newly created ``ListEntry`` instance.
+        """
+        list_content = get_object_or_404(
+            models.ListContent,
+            profile_item__topic__profile__km_user__user=self.request.user,
+            pk=self.kwargs.get('pk'))
+
+        return serializer.save(list_content=list_content)
+
+
+class ListEntryDetailView(generics.RetrieveUpdateDestroyAPIView):
```

Comment:
Since we now have this view, the `ListEntry` model should have a `get_absolute_url` method defined.